src/Twilio/Http/CurlClient.php

```diff
@@ -9,26 +9,33 @@
 class CurlClient implements Client {
     public const DEFAULT_TIMEOUT = 60;
     protected $curlOptions = [];
-    protected $debugHttp = false;
 
     public $lastRequest;
     public $lastResponse;
 
     public function __construct(array $options = []) {
         $this->curlOptions = $options;
-        $this->debugHttp = \getenv('DEBUG_HTTP_TRAFFIC') === 'true';
```

Comment:
I don't love the idea of breaking existing debugging capabilities if we can avoid it. Can we check for this env var in this client?